src/ repositories/user /UserRepository.ts

```diff
@@ -10,11 +10,21 @@ class UserRepository {
     constructor() {
         this.userModel = userModel;
     }
+    public generateObjectId() {
+        return String(Mongoose.Types.ObjectId());
+    }
 
     create = (data: any) => {
+        const userdata = {
+            _id: this.generateObjectId(),
+            ... data,
+        };
         return this.userModel.create(data);
     };
 
+    findOne = (query: any): any => {
+        return this.userModel.findOne(query);
+    }
```

Comment:
use a constant format, either use a semicolon after ending every function or don't use it at all. Don't mix and match.

Reply:
Also fix the repeating code and status codes.